src/snowflake/snowpark/dataframe.py

```diff
@@ -3401,6 +3402,15 @@ def schema(self) -> StructType:
         """
         return StructType._from_attributes(self._plan.attributes)
 
+    @cached_property
+    def dtypes(self) -> "pandas.Series":
```

Comment:
actually I feel moblize's implementation is wrong, we should return a list of tuples containing the column name and type string https://spark.apache.org/docs/3.1.2/api/python/reference/api/pyspark.sql.DataFrame.dtypes.html#pyspark.sql.DataFrame.dtypes

Reply:
Makes sense. Will change to that.